doc/managing-wallets.md

```diff
@@ -15,7 +15,7 @@ The following command, for example, creates a descriptor wallet. More informatio
 $ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true
```

Comment:
Might as well drop `descriptors=true` from this example.

Reply:
Took your advice except I left the quotes since `"wallet-01"` is quoted everywhere else in this doc.